### PR TITLE
Add release note for 1.16.2

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

**Review by ajanon — APPROVED** [review]
LGTM

File: release-notes.md [hunk 1]
```diff
@@ -3,6 +3,22 @@
...
+
+#### New features & improvements
+
+* Included Android security updates for January 2023 (see [Android Security Bulletin - January 2023](https://source.android.com/security/bulletin/2023-01-01) for more information).
+* Updated Android WebView to [108.0.5359.128](https://chromereleases.googleblog.com/2022/12/chrome-for-android-update_13.html).
+
+#### Bugs
+
+* AC-1276 finalrd.service: Failed with result 'exit-code'.
+* AC-1277 The pre-start/install hook to modify Android rootfs always failed
```
Comment (ru-fu) [inline release-notes.md:18]: Same as above.

File: announcements/1.16.2.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#### New features & improvements
+
+* Included Android security updates for January 2023 (see [Android Security Bulletin - January 2023](https://source.android.com/security/bulletin/2023-01-01) for more information).
+* Updated Android WebView to [108.0.5359.128](https://chromereleases.googleblog.com/2022/12/chrome-for-android-update_13.html).
+
+#### Bugs
+
+* AC-1276 `finalrd.service`: Failed with result 'exit-code'.
+* AC-1277 The pre-start/install hook to modify Android's root file system always failed
+* [LP #2002020](https://bugs.launchpad.net/charm-etcd/+bug/2002020): anbox-cloud-core: etcd/jammy: hook failed: "install" due to "error in Tempita setup command: use_2to3 is invalid."
```
Comment (ru-fu) [inline announcements/1.16.2.md:16]:
```suggestion
* [LP #2002020](https://bugs.launchpad.net/charm-etcd/+bug/2002020): `anbox-cloud-core`: `etcd/jammy`: hook failed: "install" due to `error in Tempita setup command: use_2to3 is invalid.`
```

File: release-notes.md [hunk 3]
```diff
@@ -3,6 +3,23 @@
 See [How to upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) or [How to upgrade the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/upgrade-anbox-cloud-appliance/24186) for
 instructions on how to update your Anbox Cloud deployment.
 
+[Details=1.16.2]
+
+## 1.16.2 (January 11 2023)
```
Comment (ru-fu) [inline release-notes.md:8]:
```suggestion
## 1.16.2 (January 12 2023)
```

File: release-notes.md [hunk 4]
```diff
@@ -3,6 +3,23 @@
...
+#### New features & improvements
+
+* Included Android security updates for January 2023 (see [Android Security Bulletin - January 2023](https://source.android.com/security/bulletin/2023-01-01) for more information).
+* Updated Android WebView to [108.0.5359.128](https://chromereleases.googleblog.com/2022/12/chrome-for-android-update_13.html).
+
+#### Bugs
+
+* AC-1276 `finalrd.service`: Failed with result 'exit-code'.
+* AC-1277 The pre-start/install hook to modify Android's root file system always failed
+* [LP #2002020](https://bugs.launchpad.net/charm-etcd/+bug/2002020): anbox-cloud-core: etcd/jammy: hook failed: "install" due to "error in Tempita setup command: use_2to3 is invalid."
```
Comment (ru-fu) [inline release-notes.md:19]:
```suggestion
* [LP #2002020](https://bugs.launchpad.net/charm-etcd/+bug/2002020): `anbox-cloud-core`: `etcd/jammy`: hook failed: "install" due to `error in Tempita setup command: use_2to3 is invalid.`
```

File: supported-versions.md [hunk 5]
```diff
@@ -2,8 +2,8 @@ Anbox Cloud currently officially supports only the most recent release. Older re
 
 To ensure you receive latest security updates and bug fixes you should upgrade to a new release of Anbox Cloud shortly after it was released.
 
-Current Release: **1.15**
-Next Release: **1.16** (scheduled for November 2022)
+Current Release: **1.16**
+Next Release: **1.17** (scheduled for February 2022)
```
Comment (ru-fu) [inline supported-versions.md:6]:
```suggestion
Next Release: **1.17** (scheduled for February 2023)
```

**Review by ru-fu — APPROVED** [review]
Will fix the last finding during publishing.

**Pull request merged**